### PR TITLE
adding in instructions around debugging a Supermarket install

### PR DESCRIPTION
**Pull request by nellshamrell**
Add in documentation around debugging a Supermarket install.

### CODE REVIEW

File: docs/CONFIGURING.md [hunk 1]
```diff
@@ -149,3 +149,22 @@ be enabled. Supermarket supports the following features.
...
 * `announcement`, Enables the foldable annoucement banner that is displayed on all views.
   when uploaded
+
+## Debugging
+
+If you are having any issues with Supermarket, you can debug it using the Rails log.  However, the way you access it in Supermarket is different from how you would access it in most Rails applications.
+
+First, you will need to change the log level to 'DEBUG'.  You do this by opening up /opt/supermarket/supermarket.rb and adding this line
+```
+  default['supermarket']['log_level'] = 'DEBUG'
```
Comment (kirtfitzpatrick) [inline docs/CONFIGURING.md]: Should this change be made in the /etc/supermarket/config.json file instead?

**Comment by nellshamrell** [comment]
Just updated the instructions...based on the other config instructions, I don't think we need to prescribe how the user sets the environmental variable, there are a few different ways to do that.  This is also keeping in line with the rest of the config docs.

**Comment by kirtfitzpatrick** [comment]
👍

**Comment by kirtfitzpatrick** [comment]
:+1:

**Pull request merged**